src/pcre2_dfa_match.c

```diff
@@ -3076,7 +3076,7 @@ for (;;)
         if (codevalue == OP_BRAPOSZERO)
           {
           allow_zero = TRUE;
-          codevalue = *(++code);  /* Codevalue will be one of above BRAs */
+          ++code;  /* The following opcode will be one of the above BRAs */
```

Comment:
Clang is correct. We are writing a value to `codevalue` which is provably never used.